Doppler.Sap/Utils/SapServiceSettings.cs

```diff
@@ -0,0 +1,16 @@
+using Doppler.Sap.Models;
+using System.Linq;
+
+namespace Doppler.Sap.Utils
+{
+    public class SapServiceSettings
+    {
+        public static SapServiceConfig GetSettings(SapConfig sapConfig, string countryCode)
+        {
+            if (sapConfig.SapServiceConfigsByCountryCode.TryGetValue(countryCode, out var serviceSettings))
+                return serviceSettings;
+
+            return null;
```

Comment:
I think that this method returns a null value, for example, in https://github.com/FromDoppler/doppler-sap/pull/34/files#diff-1f61697f85bf343beba6123463325e7eb6faa05f2b256298d24d0b15ebeb1843R54 we don't catch this exception